control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -682,6 +682,10 @@ pub struct ReplicaTopology {
     pool: Option<PoolId>,
     /// The status of the replica.
     status: ReplicaStatus,
+    /// The status of the replica.
+    child_status: Option<ChildState>,
+    /// The status of the replica.
```

Comment:
Should this be `reason for status of the replica` ?

Reply:
oops good catch, copy&pasto